### ISSUE TITLE
Error while building tflite python wheel for beaglebone black 

### ISSUE THREAD

**Issue by MTawfik93** [tensorflow/tensorflow#51616]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Debian GNU/Linux 10 (buster)
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: Beaglebone black
- TensorFlow installed from (source or binary): source
- TensorFlow version: 2.5
- Python version: 3.7
- Installed using virtualenv? pip? conda?: Created inside a virtualenv then cross compiled using cmake v3.21.1
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the problem**

**Error while building custom Tflite python wheel for Beaglebone black**

I have a beaglebone black version with cpuinfo of:

processor       : 0
model name      : ARMv7 Processor rev 2 (v7l)
BogoMIPS        : 995.32
Features        : half thumb fastmult vfp edsp thumbee neon vfpv3 tls vfpd32
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x3
CPU part        : 0xc08
CPU revision    : 2
Hardware        : Generic AM33XX (Flattened Device Tree)
Revision        : 0000
Serial          : 2033SBI01413

I want to build a custom python wheel since the prebuilt wheels aren't compatible with my board. I followed the instructions on Tensorflow lite documentation for cross compiling the py wheel for ARM devices using cmake. Link to documentation: https://www.tensorflow.org/lite/guide/build_cmake_pip

The installation is done inside a virtualenv with python version 3.7 with these packages installed inside it:
- cmake         3.21.1.post1
- numpy         1.21.2
- pip           21.2.4
- pkg_resources 0.0.0
- pybind11      2.7.1
- setuptools    57.4.0
- wheel         0.37.0

I used docker to setup a cross build environment. Then, configured custom build flags to be compatible with my target in tensorflow/lite/tools/cmake/download_toolchains.sh as instructed in the documentation. 

I entered this command:
```
tensorflow/tools/ci_build/ci_build.sh PI-PYTHON37 \
  tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh armhf
```

The Docker container environment was created but the tflite py wheel failed to compile raising this error at 96%

```
make[3]: Leaving directory '/workspace/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3.7/cmake_build'
make[2]: *** [CMakeFiles/tensorflow-lite.dir/all] Error 2
CMakeFiles/Makefile2:1145: recipe for target 'CMakeFiles/tensorflow-lite.dir/all' failed
make[2]: Leaving directory '/workspace/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3.7/cmake_build'
make[1]: *** [CMakeFiles/_pywrap_tensorflow_interpreter_wrapper.dir/rule] Error 2
CMakeFiles/Makefile2:1253: recipe for target 'CMakeFiles/_pywrap_tensorflow_interpreter_wrapper.dir/rule' failed
make[1]: Leaving directory '/workspace/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3.7/cmake_build'
Makefile:215: recipe for target '_pywrap_tensorflow_interpreter_wrapper' failed
make: *** [_pywrap_tensorflow_interpreter_wrapper] Error 2
```

**Any help to solve this issue will be greatly appreciated**

**Comment by abattery** [comment]
@terryheo for the visibility

**Comment by terryheo** [comment]
I've just tested 2.5 branch and It works well for me.

Could you share more log before the error?

**Comment by MTawfik93** [comment]
Thanks for your response. 

So I just solved the issue and it works now. The problem that I was using the configuration inside bbb_makefile.inc located in tensorflow\tensorflow\lite\tools\make\targets to target my beaglebone (This directory was deprecated last week). Apparently the Neon VFP version wasn't specified in it, so I edited ```-mfpu=neon``` to ```-mfpu=neon-vfpv3```, which compiled fine and generated the custom pywheel and confirmed that it is working. 

Here is the [Beaglebone Black TFlite Python Wheel](https://drive.google.com/file/d/1E9g4nV5f46PtaQBa_i1Xxfi1AernfhxB/view?usp=sharing) in case anyone needs it.
